packages/components/src/utils/useHovered.ts

```diff
@@ -28,8 +28,9 @@ import { RefObject, useEffect, useState } from 'react'
 import { getCurrentNode } from './getCurrentNode'
 import { useCallbackRef } from './useCallbackRef'
 
-// Get the current hover state of an element
-// Recommended to be used with useCallbackRef
+/**
+ * Get the current hover / focus state of a element of which component has no control
```

Comment:
Thank you so much for adding this. 🙏 Such a confusing hook.
```suggestion
 * Get the hover / focus state of an element over which the current component has no control
```